10.hello-docker-compose/README.md

```diff
@@ -75,7 +77,7 @@ services:
   redis:
     image: "redis:alpine"
     ports:
-      - "6379:6379"
+      - "6379:6380"
```

Comment:
is the port changed needed?

Reply:
Changed port of redis to 6380 as 6379 would most likely be already in use because of the redis container that dapr spins up during "dapr init"